docs/plugins.rst

```diff
@@ -26,12 +26,35 @@ Event Consumer
 .. autoclass:: gordon_gcp.GPSEventConsumer
     :members:
 
-Publisher
-----------
+GDNS Publisher
```

Comment:
I think it'd make more sense to divide the docs into sections for the service vs janitor. 

Relatedly, I think we should separate the plugins w/i the repo itself, e.g. `gordon_gcp.plugins.service.<plugin>` and `gordon_gcp.plugins.janitor.<plugin>`. It will avoid the name spacing issues that you encountered, and be a clearer separation (like we discussed yesterday afternoon).

Reply:
Yeah I had originally forgotten to do that.  The question is, if I do that, should I also do it for clients?  Some are specific to the janitor.  I'd prefer not, but...

Reply:
Do you mean divide each doc, or have separate docs?

Reply:
Oh, and regarding namespacing, I kinda think we should rename them anyway for clarity.  Too easy to look for a file and get the wrong one.

Reply:
We discussed on Slack.  Results: I divided the plugins doc, but the rest are ok.  I split the directories for (only) the plugins.  We're leaving the new namespacing in place.